### PR TITLE
Japan parser

### LINKED ISSUE #481: Import Japan case data
https://github.com/reustle/covid19japan-data/tree/master/docs/patient_data

### PR DESCRIPTION
**Pull request by attwad**
Taken from #730 as Anya couldn't get it to work locally, I patched and tested myself instead.

Also increases timeouts of functions because there is a lot of data in Japan and 60 secs wasn't enough.

Fixes #481

### CODE REVIEW

File: ingestion/functions/parsing/japan/japan.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                    "sourceEntryId": entry["patientId"],
+                    "sourceUrl": source_url
+                },
+                "revisionMetadata": {
+                    "revisionNumber": 0,
+                    "creationMetadata": {
+                        "curator": "auto",
+                        "date": date.today().strftime("%m/%d/%Y")
+                    }
+                },
```
Comment (axmb) [inline ingestion/functions/parsing/japan/japan.py:66]: A separate PR, but we should move this into the library. I can't recall -- is this part used, still, or does the service account identity get inferred/used based on it supplying the bearer token?

File: ingestion/functions/template.yaml [hunk 2]
```diff
@@ -25,7 +25,7 @@ Resources:
         - AWSLambdaFullAccess
         - S3WritePolicy:
             BucketName: !Ref RawSourcesBucket
-      Timeout: 60 # Seconds
+      Timeout: 600 # Seconds
```
Comment (axmb) [inline ingestion/functions/template.yaml:28]: Probably better just to remove this. I had it at 60 originally for development purposes, but in prod we probably want it to take as much time as it needs (up to the 900 second limit).

File: ingestion/functions/parsing/japan/japan.py (hunk 1 above)
Comment (attwad) [reply ingestion/functions/parsing/japan/japan.py:66]: oh I missed that, it's not needed at all, removed.

File: ingestion/functions/template.yaml (hunk 2 above)
Comment (attwad) [reply ingestion/functions/template.yaml:28]: ok fair enough, let's remove it.

**Pull request merged**